models/model.js

```diff
@@ -30,7 +30,17 @@ function selectArticle(id) {
 }
 
 function selectAllArticles(){
-    console.log('hello from model 4')
+    return db.query(`SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;`).then((response) => {
+        if (response.rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "Not Found" });
+          }
+        
```

Comment:
Fab query here. Have a think about how you can get comment count as a number. It'll make far more sense in front end!!